### ISSUE TITLE
[HOW-TO] Run a Custom Workflow

### ISSUE THREAD

**Issue by jane188** [Microsoft/EasyRepro#1157]
## Question

How to run/execute a custom workflow in UCI?

Before we used this code:
          xrmBrowser.CommandBar.ClickCommand("Run Workflow", "", true);
          xrmBrowser.Dialogs.RunWorkflow("Name of the workflow");

Now:  try to use xrmApp, and don't know how. 

Please help! 
Thanks

**Comment by AngelRodriguez8008** [comment]
Are you able to do it manually? How looks your Command Bar / Ribbon now?

**Comment by jane188** [comment]
Yes, we can run it manually.  Our workflow is under Flow -> Run Workflow (see image below). Sorry the image is not clear:
![Workflow](https://user-images.githubusercontent.com/50111593/126835989-ad665de2-0ae3-453e-b3e7-f7519ba20924.PNG)

Thank you!

**Comment by tipsey** [comment]
Were you able to find a Solution?
